src/MarginTrading.SettingsService.AzureRepositories/SimpleAzureEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.WindowsAzure.Storage.Table;
+
+namespace MarginTrading.SettingsService.AzureRepositories
+{
+    public class SimpleAzureEntity : TableEntity
```

Comment:
AzureTableEntity from Lykke.AzureStorage should be used intead of TableEntity
!!! decimal in TableEntity is not supported